src/js/iotjs.js

```diff
@@ -54,7 +54,6 @@
     try {
       var loc = '/tmp/iotjs.' + process.pid;
       lines = fs.readFileSync(loc).toString().split('\n');
-      fs.unlinkSync(loc);
```

Comment:
If we do remove this dump file, we can only get file read at the first time and failed afterwards.
Since dump file would be written continuously while new JS files are read in, it shall not be cached yet, in this case.

Reply:
This should be called when the program gets crashed, so dump file might not be used any more? Could you please point me out where we use this file in another place?

Reply:
In this implementation, stacks are generated from parser dump file with an array of frames.
Like examples I stated in issue description, it is possible that multiple error instances' stack accessed in a single run. And since the scripts are loaded with `require` continuously, the parser dump file does get new contents too.
Thus it is required to not remove the dump file to get error stacks in newly loaded scripts be generated correctly.

Reply:
Ok, looks good to me now, and only one question we need to acquire, when do we clean this dump file up, in our past case, @lolBig give a feedback about this, when the program exits and restarts in a infinite loop, the dump file will be generated every circle, that eats the disk up to empty.